app/__init__.py

```diff
@@ -809,10 +841,6 @@ def add_review():
     review.images.append(image)
 
   db.session.add(review)
-  
-  spot = Spot.query.filter_by(id=beach_id).first()
```

Comment:
did you mean to delete this?

Reply:
nvm, seems like you moved it higher in the file

Reply:
Yea I just moved it up so I could feed it into the email variables.